fake-ipa/Run-env/04-start-minikube.sh

```diff
@@ -0,0 +1,16 @@
+set -e
+
+# Start Minikube with insecure registry flag
+minikube start --insecure-registry 172.22.0.1:5000
+
+# SSH into the Minikube VM and execute the following commands
+sudo su -l -c "minikube ssh sudo brctl addbr ironicendpoint" "${USER}"
+sudo su -l -c "minikube ssh sudo  ip link set ironicendpoint up" "${USER}"
+sudo su -l -c "minikube ssh sudo  brctl addif  ironicendpoint eth2" "${USER}"
+sudo su -l -c "minikube ssh sudo  ip addr add 172.22.0.2/24 dev ironicendpoint" "${USER}"
+
+# Firewall rules
+for i in 8000 80 9999 6385 5050 6180 53 5000; do sudo firewall-cmd --zone=public --add-port=${i}/tcp; done
+for i in 8000 80 9999 6385 5050 6180 53 5000; do sudo firewall-cmd --zone=libvirt --add-port=${i}/tcp; done
```

Comment:
same, and also can we not use the previous loop for both commands ? they iterate over same ports